Sources/PluginLibrary/NamingUtils.swift

```diff
@@ -189,10 +189,11 @@ fileprivate func sanitizeTypeName(_ s: String, disambiguator: String) -> String
     // conflict.  This can be resolved recursively by stripping
     // the disambiguator, sanitizing the root, then re-adding the
     // disambiguator:
-    let e = s.index(s.endIndex, offsetBy: -disambiguator.characters.count)
-    #if swift(>=4.0)
+    #if swift(>=3.2)
```

Comment:
fyi - style wise, this one is the only one you are adding that is indented.  I haven't checked what we did in the other case.

Reply:
Looks like there are other places where we currently indent these conditionals.  So I'll leave it as-is for now.